pennylane/ops/op_math/__init__.py

```diff
@@ -73,6 +73,9 @@
 .. autosummary::
     :toctree: api
 
+    ~decompositions.one_qubit_decomposition
```

Comment:
To get it to show as `qml.ops.one_qubit_decomposition`, I guess it's the familiar solution of changing the current module for this section?

Reply:
It's confusing though that we have `qml.ops.op_math` in the API sidebar, but no `qml.ops`. I believe this will be the first time we have something 'user facing' under `qml.ops` directly

Reply:
Yes that's a good point :thinking: We could create a `qml.ops` docs page to contain, e.g., the content currently in `qml.ops.op_math` that is accessible at top level. Not sure if we want to do it in this PR though.

Reply:
I separated decompositions and control decompositions. I think the original problem is to have an op math page instead of ops. We already have situation in this page where we show top level docs like adjoint and control. Also the whole module is top level imported.

As a temporary solution I separated decompositions and control decompositions, the link for decompositions is `qml.ops`. Let me know what you think?